src/wallet/wallet.cpp

```diff
@@ -2590,28 +2590,28 @@ std::shared_ptr<CWallet> CWallet::Create(WalletContext& context, const std::stri
         }
     }
 
-    if (!gArgs.GetArg("-addresstype", "").empty()) {
-        std::optional<OutputType> parsed = ParseOutputType(gArgs.GetArg("-addresstype", ""));
+    if (!context.args->GetArg("-addresstype", "").empty()) {
```

Comment:
In commit "Use `context.args` in `CWallet::Create` instead of `gArgs`." (1153ad33ffc821917c76777d971f0074f7af7590)

Just as a style suggestion I might write:

```c++
ArgsManager& args = *Assert(context.args);
```

above and use `args` instead of `context.args` throughout this function for less verbosity and for a clearer assert error.

Reply:
Thank you 👍, I had something similar in mind.